spec/lib/twingly/url_spec.rb

```diff
@@ -102,6 +114,16 @@ def valid_urls
       end
     end
 
+    context "IDNA 2003 vs IDNA 2008" do
```

Comment:
context and its does not really follow suit with rest of the spec file. How about something like:

```ruby
context "when given IDN URL containing ß" do # could generalize but it's early in the morning and brain is slow
 ...
  it "should conform to the IDNA 2008 standard" do
    ...
  end
end
```

Should also be placed under the describe for `#normalized`, see for instance https://github.com/twingly/twingly-url/blob/77631f74eb5f52e42e009537d0963eaea33ac0a8/spec/lib/twingly/url_spec.rb#L388